src/mlservice.h

```diff
@@ -158,7 +158,16 @@ namespace dd
       _init_parameters = ad.getobj("parameters");
       this->_inputc.init(_init_parameters.getobj("input"));
       this->_outputc.init(_init_parameters.getobj("output"));
-      this->init_mllib(_init_parameters.getobj("mllib"));
+      try
```

Comment:
Do you need this ? The main principle has always been to relay lower-level exceptions with their messages, and not obfuscate them at higher levels, when possible.

Reply:
I put this here because exception do not seem to be catched at upper level:  dede terminates in case of failure of init_mllib

Reply:
Let's make this another issue, all exceptions should be caught at upper levels.

Reply:
removed